upgrade/steps_test.go

```diff
@@ -39,9 +41,20 @@ func TestGetWorkingBranch(t *testing.T) {
 				UpgradeBridgeVersion: true,
 				TargetBridgeRef:      &Version{SemVer: semver.MustParse("v1.2.3")},
 			},
+			branchSuffix:        "",
 			targetBridgeVersion: &Version{SemVer: semver.MustParse("1.2.3")},
 			expected:            "upgrade-pulumi-terraform-bridge-to-1.2.3",
 		},
+		{
+			c: Context{
+				UpgradeBridgeVersion: true,
+				TargetBridgeRef:      &Version{SemVer: semver.MustParse("v1.2.3")},
+				PRTitlePrefix:        "foo",
```

Comment:
What is a realistic PR title prefix? What is a PR title prefix?

Reply:
This is here right? 

https://github.com/pulumi/upgrade-provider/blob/03d6e987a846ff0b8d848f02a0abd1476aee4efd/main.go#L256

Reply:
Can you put some checks for real values.
```
""
"[TEST]"

```
Presumably `[DOWNSTREAM TEST][PLATFORM]` is a prefix too? 

https://github.com/pulumi/pulumi-aws/pull/5169

Just to make sure it's sane.